Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -686,8 +686,11 @@ private static void RegisterInstance(MixedRealityToolkit toolkitInstance)
                 // Set the instance to active
                 MixedRealityToolkit.activeInstance = toolkitInstance;
                 toolkitInstances.Add(toolkitInstance);
```

Comment:
Per https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4381, given that the same instance is being initialized multiple times, does this also imply that toolkitInstances will have a duplicate (i.e. two of the same instances?)

Reply:
We're using a hashset to avoid duplicates.